src/Sentry/SentryFailedRequestHandler.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Sentry.Protocol;
+
+namespace Sentry
+{
+    internal class SentryFailedRequestHandler : ISentryFailedRequestHandler
+    {
+        private readonly IHub _hub;
+        private readonly SentryOptions? _options;
```

Comment:
```suggestion
        private readonly SentryOptions _options;
```

I don't think this class can operate without options.  Let's make it non-nullable.  Adjust throughout please.